presto-native-execution/pom.xml

```diff
@@ -156,6 +156,11 @@
             <groupId>com.facebook.airlift</groupId>
             <artifactId>log-manager</artifactId>
         </dependency>
+        <dependency>
```

Comment:
This can be reverted after the test utility functions are moved to `AbstractTestQueryFramework.java`.

Reply:
Because the test utility function was overridden , had to keep this change.